src/data/UserFunctions.js

```diff
@@ -119,8 +119,9 @@ export const allComments = () => {
 };
 
 export const getDataforUserSearch = param => {
```

Comment:
Try not to use generic words like "param" or "val" as param names,"query" would be a more descriptive name

Reply:
getDataForUserSearch